CommonTools/PileupAlgos/python/customizePuppiTune_cff.py

```diff
@@ -26,11 +26,25 @@ def UpdatePuppiTuneV14(process):
   del process.updatedPatJetsPuppiJetSpecific
   process.puppiSequence = cms.Sequence(process.puppiMETSequence+process.fullPatMetSequencePuppi+process.patPuppiJetSpecificProducer+process.slimmedJetsPuppi)
   #
-  # Adapt for PUPPI tune V14
+  # Adapt for PUPPI tune V15
   #
   process.puppi.PtMaxCharged = 20.
   process.puppi.EtaMinUseDeltaZ = 2.4
   process.puppi.PtMaxNeutralsStartSlope = 20.
+  process.puppi.NumOfPUVtxsForCharged = 2
+  process.puppi.algos[0].etaMin = [-0.01]
   process.puppiNoLep.PtMaxCharged = 20.
   process.puppiNoLep.EtaMinUseDeltaZ = 2.4
   process.puppiNoLep.PtMaxNeutralsStartSlope = 20.
+  process.puppiNoLep.NumOfPUVtxsForCharged = 2
+  process.puppiNoLep.algos[0].etaMin = [-0.01]
+
+  from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+  phase2_common.toModify( process.puppi, EtaMinUseDeltaZ = 4.0)
+  phase2_common.toModify( process.puppiNoLep, EtaMinUseDeltaZ = 4.0)
```

Comment:
this part does not match the master version, where phase2 (either default or with this custom function) would give `NumOfPUVtxsForCharged = 0, EtaMinUseDeltaZ = -1.`. A consistent behavior should be provided (either the default master version for phase2 would need to change or this method be consistently updated.

BTW, this `UpdatePuppiTuneV15` is itself pretty confusing because it still contains "fake" overrides to values which are already set to V14 values by calling `makePuppiesFromMiniAOD`, which loads the release default.
Some cleanup would be useful so that only values that need changing are actually explicitly changed.

Reply:
@ahinzmann 
please clarify on the status of following up on the code review for this PR.
Thank you.

Reply:
The idea is to some Phase-2 plots with this recipe first, before actually changing the master. So this PR is on hold as well then.